tests/unit/dojson/test_dojson_institutions.py

```diff
@@ -884,38 +846,32 @@ def test_related_institutes_from__510_a_w_0_successor():
         '</datafield>'
     )  # record/902831
 
-    expected = [
-        {
-            'curated_relation': True,
-            'name': 'HZB, Berlin',
-            'relation_type': 'successor',
-            'record': {
-                '$ref': 'http://localhost:5000/api/institutions/911753',
-            },
-        },
-    ]
     result = institutions.do(create_record(snippet))
 
-    assert validate(result['related_institutes'], subschema) is None
-    assert expected == result['related_institutes']
+    assert 'related_records' not in result
 
 
-def test_related_institutes_from__invalid_510__0():
-    schema = load_schema('institutions')
-    subschema = schema['properties']['related_institutes']
+def test_related_records_from_510__a_w_0_discards_other():
```

Comment:
this should be preserved, with `other` part of `relation_freetext`. The only reason for discarding relations (for now) is for relations going in the wrong direction, which require a solution to #2433.